daml-lf/language/daml-lf.bzl

```
@@ -88,17 +88,21 @@ LF_VERSIONS = [
     "1.12",
     "1.13",
     "1.14",
+    "1.15",
     "1.dev",
 ]
 
-# All LF versions for which we have protobufs.
-PROTO_LF_VERSIONS = [ver for ver in LF_VERSIONS if versions.gte(ver, "1.14")]
+# Next version of LF. not fully functional in general.
+lf_version_next = "1.15"
 
-# We support older LF versions using an older compiler binary
-LEGACY_COMPILER_LF_VERSIONS = [ver for ver in LF_VERSIONS if versions.lte(ver, "1.13")]
+# All LF versions for which we have protobufs.
+PROTO_LF_VERSIONS = [ver for ver in LF_VERSIONS if versions.gte(ver, "1.14") if ver != lf_version_next]
 
 # The subset of LF versions accepted by the compiler in the syntax
 # expected by the --target option.
-COMPILER_LF_VERSIONS = [ver for ver in LF_VERSIONS if ver not in LEGACY_COMPILER_LF_VERSIONS]
+COMPILER_LF_VERSIONS = [
+    "1.14",
+    "1.dev",
+]
```

following on @cocreature's comment above, we might want this here
```suggestion
COMPILER_LF_VERSIONS = PROTO_LF_VERSIONS
```
I'm not sure if these are ever different, though

---

PROTO_LF_VERSIONS and COMPILER_LF_VERSIONS are not related. 

The first one is the set of protobuf snapshot we have. A snapshot is a copy of the dev proto at the time we release the LF version, so client (e.g Hub) can use it without being annoy by the change we make to the dev proto. 

The latter is the set of version output by the compiler (the compiler like the engine always use the dev proto). 

We have no reason to bind the two. 


I am fine to set up both as range with filter though,